python/grass/temporal/temporal_topology_dataset_connector.py

```diff
@@ -43,8 +43,6 @@ class TemporalTopologyDatasetConnector:
     - finishes
     - finished
 
-    .. code-block:: python
-
```

Comment:
I don't get the change here. I think this should stay here. The rendered doc is here: [grass.osgeo.org/grass-stable/manuals/libpython/grass.temporal.html](https://grass.osgeo.org/grass-stable/manuals/libpython/grass.temporal.html#grass.temporal.temporal_topology_dataset_connector.TemporalTopologyDatasetConnector)